mkdocs/docs/tutorials/traffic-engineering/session-affinity.md

```diff
@@ -0,0 +1,179 @@
+---
+template: main.html
+---
+
+# Session Affinity
+
+!!! tip "Scenario: A/B testing with session affinity (sticky sessions)"
+
+    [Session affinity](../../../concepts/buildingblocks/#traffic-engineering) ensures that the version to which a particular user's request is routed remains consistent throughout the duration of the experiment. In this tutorial, you will use a [FixedSplit deployment](../../../concepts/buildingblocks/#deployment-patterns) in conjunction with session affinity as depicted below.
+
+    ![Session affinity](../../images/session-affinity-exp.png)
+
+
+???+ warning "Before you begin, you will need... "
+    > **Note:** Please choose the same K8s stack (for example, Istio, KFServing, or Knative) consistently throughout this tutorial. If you wish to switch K8s stacks between tutorials, start from a clean K8s cluster, so that your cluster is correctly setup.
+
+## Steps 1 to 3
+    
+Please follow steps 1 through 3 of the [quick start tutorial](../../../getting-started/quick-start/#1-create-kubernetes-cluster).
```

Comment:
We already remind the users to pick the same k8s stack. But I still didn't know which when I got to the quick start page. Perhaps before Step 1 we should list the available k8s stacks for this tutorial. Then I will be conscious of my choice before I switch context.

Reply:
I like the change here. Very clear.